projects/dotcms-field-elements/src/components/dot-date-time/dot-date-time.tsx

```diff
@@ -191,6 +193,8 @@ export class DotDateTimeComponent {
                         <label>
                             {this.timeLabel}
                             <dot-input-calendar
+                                aria-describedby={getHintId(this.hint)}
```

Comment:
Not here. The user will hear it the first time when he focus the first `dot-input-calendar`